cli/azd/pkg/executil/executil.go

```diff
@@ -197,10 +198,31 @@ type RunArgs struct {
 	EnrichError bool
 }
 
+type RedactData struct {
```

Comment:
Do we need to export this type?

```suggestion
type redactData struct {
```